src/components/tableview/TableTable.vue

```diff
@@ -210,8 +211,12 @@ export default {
           cellEditCancelled: cell => cell.getRow().normalizeHeight(),
           formatter: (cell) => _.isNil(cell.getValue()) ? '(NULL)' : cell.getValue(),
           editorParams: {
-            verticalNavigation: useTextarea ? 'editor' : undefined,
+            verticalNavigation: useVerticalNavigation ? 'editor' : undefined,
             search: true,
+            ...(column.dataType === 'bool'
+              ? { values: ['true', 'false'] }
```

Comment:
should these have quotes around them, or just be `[true, false]`?

Reply:
We probably need an entry in `data_mutators.js` for booleans to not turn them into strings too

Reply:
I think tabulator accept only strings as values. Btw it's working.